library-client-rtsp/src/main/java/com/alexvas/rtsp/codec/VideoDecodeThread.kt

```diff
@@ -16,11 +18,11 @@ class VideoDecodeThread (
         private val videoFrameQueue: FrameQueue,
         private val onFrameRenderedListener: OnFrameRenderedListener) : Thread() {
 
-    private var isRunning = true
+    private var exitFlag: AtomicBoolean = AtomicBoolean(false);
 
     fun stopAsync() {
         if (DEBUG) Log.v(TAG, "stopAsync()")
-        isRunning = false
+        exitFlag.set(true);
```

Comment:
Redundant semicolom

Reply:
thanks - you can tell I'm a Java developer and haven't worked with Kotlin much yet :)